index.html

```diff
@@ -51,21 +51,24 @@ <h2>A personal list of curious psychological phenomena, paradoxes and cognitive
       </div>
 
       <section class="introduction">
-        <p>So, this is just a small personal project that came to being because of an interest 
-          on random things and because I wanted to put to practice my recently acquired basic 
-          knowledge on HTML, CSS and JavaScript.
-          That said, I know there's hundreds of paradoxes and phenomenons like the ones included 
-          on this page, but if I wanted to include them all it would just be a thorough exhaustive 
-          list without cause or meaning. Instead, I detailed only the ones that I personally had 
-          fun finding out with, that have a specific usefulness for me or that woke my curiosity 
-          for whatever reason. Some of them actually let me view personal situations with a new 
-          lens or perspective.
+        <p>So, this is just a small personal project that came to being because of an interest on random
+          things and because I wanted to put to practice my recently acquired basic knowledge on HTML, CSS
+          and JavaScript. That said, I know there's hundreds of paradoxes and phenomenons like the ones
+          included on this page, but if I wanted to include them all it would just be a thorough exhaustive
+          list without cause or meaning. Instead, I detailed only the ones that I personally had fun finding
+          out with, that have a specific usefulness for me or that woke my curiosity for whatever reason.
+          Some of them actually let me view personal situations with a new lens or perspective.
         </p>
         <p>
-          However, I would also like for this list to be collaborative and 
-          I'm always interesting in curious facts in general, so if you can think of a phenomenon 
-          that really interests you or that you somehow applied to your day-to-day life please let me 
-          know about it!.
+          However, I would also like for this list to be collaborative and I'm always interesting in curious
+          facts in general, so if you can think of a phenomenon that really interests you or that you somehow
+          applied to your day-to-day life please let me know about it!.
+        </p>
+        <br>
+        <p class="italics disclaimer">
+          *Disclaimer: the inclussion of these biases does not mean that I approve the experiments that might
+          have been conducted in order to discover them nor the original use that they were given, which in
+          some cases might have ben realted to economical or political interests. 
```

Comment:
Great disclaimer! :ok_hand:

Reply:
I figured it was less code on the html file and cleaner to just add the id to the OH, FATHER OF ALL <html> element and not adding a class to all the specific elements I want to select. So instead of: 
```
<main class="contact>
<h2 class="contact">
```
 &
```
main .contact {}
h2 .contact {}
```

I can go: 
`<html id="contact">`
&
```
#contact main {}
#contact h2 {}
```

It also makes more sense to me that in the subpages the #id comes before the element, for faster reading. What do you think?